.github/workflows/on_pull_request.yml

```diff
@@ -151,60 +103,25 @@ jobs:
         with:
           name: compiled-js-and-types
           path: packages/
+          if-no-files-found: error
 
-      - name: Run eslint
-        run: yarn lint
-
-  tests:
-    name: Test repository using React 17
-    needs: build
-    runs-on: ubuntu-latest
-    steps:
-      - name: Checkout Repo
-        uses: actions/checkout@v2
-
-      - name: Setup Node.js 12.x
-        uses: actions/setup-node@v2
-        with:
-          node-version: 12.x
-
-      - name: Get yarn cache directory path
-        id: yarn-cache-dir-path
-        run: echo "::set-output name=dir::$(yarn config get cacheFolder)"
-
-      - name: Load Yarn cache
-        uses: actions/cache@v2
-        id: yarn_cache_id
-        with:
-          path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
-          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
-          restore-keys: |
-            ${{ runner.os }}-yarn
+      - name: Build website
+        run: yarn build:website
+        env:
+          AIRTABLE_APIKEY: ${{ secrets.AIRTABLE_APIKEY }}
+          AIRTABLE_BASEID: ${{ secrets.AIRTABLE_BASEID }}
+          GATSBY_DOCSEARCH_APIKEY: ${{ secrets.GATSBY_DOCSEARCH_APIKEY }}
 
-      - name: Node modules cache
-        uses: actions/cache@v2
-        id: node_modules_cache_id
+      - name: Store build artifacts
+        uses: actions/upload-artifact@v2
         with:
+          name: compiled-website
           path: |
-            node_modules
-            */*/node_modules
-          key: ${{ runner.os }}-${{ hashFiles('**/yarn.lock') }}
-
-      - name: Install Dependencies
-        if: steps.yarn_cache_id.outputs.cache-hit != 'true' || steps.node_modules_cache_id.outputs.cache-hit != 'true'
-        run: yarn install --immutable
+            packages/paste-website/.cache/mappings.json
```

Comment:
TBH not sure if we actually need this, but it threw an error when I omitted all of the `.cache` files.

Reply:
We need this file. I might refactor it later, but this file needs to be commit and shipped.

Reply:
Yeah that sounds reasonable. FWIW this isn't being used to build the website on release, only for the pull requests CI.